veejay-current/veejay-server/veejay/vj-perform.c

```diff
@@ -771,15 +771,15 @@ static int vj_perform_record_buffer_init()
 
 static void vj_perform_record_buffer_free()
 {
-
-    if(record_buffer->Y) free(record_buffer->Y);
-    record_buffer->Y = NULL;
-    if(record_buffer->Cb) free(record_buffer->Cb);
-    record_buffer->Cb = NULL;
-    if(record_buffer->Cr) free(record_buffer->Cr);
-    record_buffer->Cr = NULL;
-    if(record_buffer)
+    if(record_buffer){
```

Comment:
vj_perform_record_buffer_free is only called from vj_perform_free, record_buffer must exist when it enters this function (why this change?)

Reply:
This fix a segfault (SIGSEGV Invalid memory reference)
how to reproduce : veejay -g file-that-doesnt-exist.ext